.eslintignore

```diff
@@ -0,0 +1,5 @@
+**/node_modules/*
```

Comment:
You didn't create the lab-<NAME> directory. It also looks like you are working from the master branch. You have to create a new branch/folder. Feel free to ask me or Duncan if you need help with that :)